tests/dialects/test_tsql.py

```diff
@@ -261,7 +260,8 @@ def test_tsql(self):
         self.validate_identity("SELECT * FROM ##foo")
         self.validate_identity("SELECT a = 1", "SELECT 1 AS a")
         self.validate_identity(
-            "DECLARE @TestVariable AS VARCHAR(100)='Save Our Planet'", check_command_warning=True
+            "DECLARE @TestVariable AS VARCHAR(100) = 'Save Our Planet'",
+            # check_command_warning=True
```

Comment:
Since we don't parse `DECLARE` as a command anymore, why keep this commented? Just get rid of it.